examples/oic.r.amacl-example.json

```diff
@@ -1,16 +1,14 @@
 {
   "resources": [
     {
-      "href": "coaps://IP-ADDR/temp",
-      "rel": "some-rel",
-      "rt": "oic.r.temperature",
-      "if": "oic.if.a"
+      "href": "/temp",
+      "rt": ["oic.r.temperature"],
+      "if": ["oic.if.a"]
```

Comment:
If you're going to use examples, they should probably be accurate.  Since all Resources must include oic.if.baseline, you should add it to all examples like this:
"if": ["oic.if.a", "oic.if.baseline"]